### PR TITLE
Document new wrapper for overlay options to rpi_camera

### PR DESCRIPTION
**Pull request by alxrdn**
## Proposed change

Add documentation for two new configuration variables added to `rpi_camera`.

Add wrapper for overlay options, in order to be able to add date/time (for example) on the pictures.
Add two new configuration variables :
- overlay_metadata
- overlay_timestamp

Their formats are documented at [`--annotate,    -a        Enable/set annotate flags or text`](https://www.raspberrypi.org/documentation/raspbian/applications/camera.md).

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: [add overlay options wrapper to rpi_camera
#34461](https://github.com/home-assistant/core/pull/34461)
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

**Comment by alxrdn** [comment]
> It seems that this PR is targeted against an incorrect branch. Documentation updates which apply to our current stable release should target the `current` branch. Please change the target branch of this PR to `current` and rebase if needed. If this is documentation for a new feature, please add a link to that PR in your description.

I probably should have done a draft PR so I could have submitted this PR from the beginning with the link to `code` matching PR.

Afaik, this PR is targeted to the correct branch `next` as it relates to a new feature I am adding in the related `code` PR.

File: source/_integrations/rpi_camera.markdown [hunk 1]
```diff
@@ -16,10 +16,16 @@ To enable this camera in your installation, add the following to your `configura
 
 ```yaml
 # Example configuration.yaml entry
-camera:
-  - platform: rpi_camera
+rpi_camera:
+    image_width: 1296
```
Comment (frenck) [inline source/_integrations/rpi_camera.markdown]: We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.

**Review by frenck — DISMISSED** [review]
Thanks, for the PR @alxrdn!

Left a comment, could you please take a look?

Thanks! 👍

File: source/_integrations/rpi_camera.markdown (hunk 1 above)
Comment (alxrdn) [reply source/_integrations/rpi_camera.markdown]: no problem, I removed optional parameters from the example

**Review by fabaff — APPROVED** [review]

**Pull request merged**